modules/lang/en.lang.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Future task: Can we use GPT to get the translations automatically? or via script? lol will be hard with 5 languages